src/routes/dashboard/csv/downloadCSV.ts

```diff
@@ -0,0 +1,115 @@
+import type { Questions } from '../types'
+
+export function toCSV(channels: string[], questions: Map<string, Questions>) {
+  let csv =
+    'Channel,Number of Questions, Answered by Staff, Answered by Community, Unanswered,'
+
+  const sortedQs: Map<string, Map<string, Questions>> = reshape(
+    channels,
+    questions
+  )
+
+  const [first] = sortedQs.values()
+  if(!first) return /** @todo throw some error */
```

Comment:
```suggestion
  if (!first) return /** @todo throw some error */
```

🤔 prettier should have caught this